README.md

```diff
@@ -15,15 +15,21 @@ Blur images in the WordPress editor interface if they don't have alt text. Socia
 
 This plugin blurs images in the WordPress Block Editor interface if there is no alt text on the image, or if the alt text contains .jpg, .png, or .gif. **Site visitors** still see the image just fine, but **editors** will see a blurred image in the dashboard. Hovering or selecting the image removes the blur effect.
 
+As of v0.93, this plugin can optionally blur images on the front end **only for logged-in users with the `edit_pages` capability**.
+
 Most styles work in all "modern browsers" (not Internet Explorer). Media & Text blocks with the "crop" setting only work in very recent browsers that support the `:has()` psuedo selector.
 
 ### Note: Unblurring Decorative Images
 
 Note: This will blur __all__ images in the editing UI that don't have alt text — even decorative images that should not have alt text, or images that are described elsewhere in the text. If you have such an image, you can give the img element the class `noalt` and it will not blur.
 
+On the front-end, images that are marked with `[aria-hidden="true"]` (or are a descendent of an element with that attribute) will not be blurred.
+
 ### Settings
 
-This plugin creates a "Blur No-Alt Message Display" options screen in the "Settings" with one option to toggle the informative message at the top of the block editor screen.
```

Comment:
I'd like to keep some text here, explaining that there is an options screen under Settings. (My style is more conversational. I get that it's nice to write short, but it can be gatekeepy for people who aren't very experienced.) In this case, it might seem to be enough to just have this under "Settings", but in my experience, novice users don't necessarily know where to look to find the options screen. How about including something like this:

The "Blur No-Alt Message Display" options screen in the Settings menu allows you to:

Reply:
I get it -- there's so much pressure to be succinct. As though we're still using 300 baud modems, and paying for every keystroke.

But it's super gatekeepy. I hope to *teach* people with doc. I have, for my entire career, stumbled over people's "succinct" instructions, which are typically either too ambiguous or too esoteric to understand without context. I love my users!

Reply:
New language:

> ### Settings
> 
> This plugin's settings are found in "Settings" > "Blur No-Alt" in the WordPress admin menu. The plugin provides two settings for customizing the plugin's behavior:
> 
> - Toggle the informational message about blurred images above the title in the block editor
> - Toggle whether logged-in users see blurred images when viewing the front-end of the site
> 
> When the front-end styles are enabled, developers can customize which users see the blurred images with the `blur_no_alt_front_end_blur_capability` filter (default: `edit_pages`)